### DISCUSSION TITLE
Add some readme with basics instructions, examples etc

### DISCUSSION THREAD

**Discussion by micwoj92** [DcodingTheWeb/EasyCodeIt#9] [Ideas]
Couple points as example
1. Building instructions, for example:
`git clone https://github.com/DcodingTheWeb/EasyCodeIt.git`
`cd EasyCodeIt`
`cmake -B build`
`cmake --build build`

2. Examples how to use
3. Current compatibility with AutoIt
4. Roadmap

**Comment by TheDcoder** [comment]
Thanks for showing interest in the project, currently it is in very early stages and currently only comprises of a lexical scanner/tokenizer.

I will add build instructions, examples etc. as the project progresses, it's too early to do that right now. You can check the [project's thread](https://www.autoitscript.com/forum/topic/204477-easycodeit-cross-platform-autoit-implementation) at the AutoIt forum to see the current status and roadmap.
